modules/contact-form/grunion-contact-form.php

```diff
@@ -2007,9 +2007,16 @@ function process_submission() {
 			$reply_to_addr = $comment_author_email;
 		}
 
+		/*
+		 * Build the message headers
+		 *
+		 * We don't need to specify a Content-Type header, because PHPMailer will automatically generate the
+		 * proper content-type for each part of the message once it detects that an AltBody is added.
+		 *
+		 * wp_mail() automatically sets the Charset to the site's charset, so we don't need to do that either.
+		 */
 		$headers = 'From: "' . $comment_author . '" <' . $from_email_addr . ">\r\n" .
-					'Reply-To: "' . $comment_author . '" <' . $reply_to_addr . ">\r\n" .
-					'Content-Type: text/html; charset="' . get_option( 'blog_charset' ) . '"';
```

Comment:
I don't think so. Grunion isn't setting the content type anymore, it's relying on PHPMailer to do it. It seems like adding code to Grunion to set the type would be redundant and possible lead to errors in the future.